src/material-experimental/mdc-chips/chips.scss

```diff
@@ -100,6 +100,19 @@ input.mat-mdc-chip-input {
   flex: 1 0 $mat-chip-input-width;
 }
 
+// The margin value is set in MDC.
+$mat-mdc-chip-margin: 4px;
+
+// Don't let the chip margin increase the mat-form-field height.
+mat-chip-grid {
```

Comment:
Should we target the `mat-mdc-chip-grid` class here? In general we've been avoiding tag selectors.

Reply:
Good point! Done.